.github/workflows/composer-normalize.yml

```diff
@@ -0,0 +1,28 @@
+name: normalize composer.json
+
+on:
+  push:
+    paths:
+      - 'composer.json'
+  pull_request:
+    paths:
+      - 'composer.json'
+
+jobs:
+  normalize:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+
+      - name: Validate Composer configuration
+        run: composer validate --strict
+
+      - name: Normalize composer.json
+        run: |
+          composer global require ergebnis/composer-normalize
+          composer normalize
+
+      - uses: stefanzweifel/git-auto-commit-action@v4.0.0
```

Comment:
Will this work on PRs made from forks?

I read https://github.com/stefanzweifel/git-auto-commit-action/issues/25 but am unsure what the outcome here would be.

Reply:
You are right, it will depend on the permission settings in the fork.
I will remove the `pull_request` trigger and run it only on master branch on push!?

Reply:
https://github.com/barryvdh/laravel-ide-helper/pull/959/commits/0af2ec3af3131a234b071885da1eb6e982ed630e